src/shell/plugins/Makefile.am

```diff
@@ -4,7 +4,9 @@ AM_CFLAGS = \
 AM_LDFLAGS = \
 	$(CODE_COVERAGE_LIBS)
 AM_CPPFLAGS = \
-	-I$(top_srcdir)
```

Comment:
Commit message typo: `wtih`

Also, something to be aware of is that the new `flux/shell.h` changes _require_ that `FLUX_SHELL_PLUGIN_NAME` be set before including `shell.h` (The commit message here makes it sound like it is optional, which maybe it should be?)

Reply:
Thanks, I reworded that commit to:
```
build: define FLUX_SHELL_PLUGIN_NAME
    
Problem: an upcoming change to shell logging will require
FLUX_SHELL_PLUGIN_NAME to be defined before including shell.h.
    
Add definition to AM_CPPFLAGS.
```